lib/GhcideSteal.hs

```diff
@@ -10,24 +10,32 @@ import Control.Monad.Trans.Class
 import Control.Monad.Trans.Maybe
 import Data.Array
 import Data.Containers.ListUtils (nubOrd)
+import Data.Foldable (foldl')
 import Data.List (isSuffixOf)
 import qualified Data.Map as M
 import Data.Maybe (catMaybes)
+import qualified Data.Set as S
 import qualified Data.Text as T
-import DynFlags
-import FastString (unpackFS)
-import qualified FastString as FS
 import GHC
+import GHC.Data.FastString (unpackFS)
+import qualified GHC.Data.FastString as FS
+import GHC.Iface.Ext.Types
+import GHC.Iface.Ext.Utils
+import GHC.Plugins hiding ((<>))
+import GHC.Types.Error
+import GHC.Utils.Outputable hiding ((<>))
 import HieDb hiding (pointCommand)
-import HieTypes
-import HieUtils
+--import HieTypes
+--import HieUtils
 import Language.LSP.Types
-import MonadUtils (mapMaybeM)
-import Name
-import Outputable hiding ((<>))
-import SrcLoc
 import System.FilePath ((</>))
 
+mapMaybeM :: Monad m => (a -> m (Maybe b)) -> ([a] -> m [b])
```

Comment:
Would this do the same thing as `\f -> fmap catMaybes . traverse f`?

Reply:
Yes I think so.